firedrake/mg/interface.py

```diff
@@ -91,8 +91,8 @@ def restrict(fine_dual, coarse_dual):
         if len(Vc) != len(Vf):
             raise ValueError("Mixed spaces have different lengths")
         for in_, out in zip(fine_dual.split(), coarse_dual.split()):
-            _, myrestrict, _ = firedrake.dmhooks.get_transfer_operators(in_.function_space().dm)
-            myrestrict(in_, out)
+            manager = firedrake.dmhooks.get_transfer_manager(in_.function_space().dm)
```

Comment:
I think the way things are now coded we can get the transfer manager once for the outer space (because there is really only one). But maybe not, and this is safer.